storage_service/locations/models/package.py

```diff
@@ -486,7 +486,10 @@ def extract_file(self, relative_path='', extract_path=None):
 
         # The basename is the base directory containing a package
         # like an AIP inside the compressed file.
-        basename = self.get_base_directory()
+        try:
+            basename = self.get_base_directory()
+        except CalledProcessError:
```

Comment:
Since this is a problem generated inside get_base_directory, it makes more sense to handle the exception (or not) there.